.gitignore

```diff
@@ -1,8 +1,14 @@
-# ignore misc build/test files
-.env-lint
+# ignore uv venv
+.venv
+
+# ignore standard cache folders
+.*_cache
+
+# ignore built files
+hwbench.egg-info
```

Comment:
Are you sure this is needed with uv build ?

Reply:
I had them in my local repo even working with uv, so I would say that they still can get into there